### PR TITLE
fix(statusProfileNavBarTabButton): Fixed backgound color and ring for…

### LINKED ISSUE #15479: Missing color and ring on the profile icon in the bottom left corner
# Bug Report

## Description

image

## Steps to reproduce

1. Open the app

#### Expected behavior

The icon has the same color as the popup and the ring 

#### Actual behavior

Color is black

### Additional Information

- Status desktop version: master `06184e4d955269c8cfa5163a07d0bf1bb8d5dbd3`
- Operating System: Ubuntu 22

### PR DESCRIPTION
**Pull request by Seitseman**
… StatusNavBarTabButton that's displayed at left bottom of the main window.

Fixed signal typo in CommunityTokensStore.
Closes #15479.

### What does the PR do

it looks like name clashing in AppMain.qml, specifically for QML Singleton Utils (at \ui\imports\utils\qmldir) and StatusQ.Core.Utils.
Fixed by adding a Qualifier to qml singleton import statement: `import utils 1.0 as QmlUtils`

### Affected areas
AppMain.qml

https://github.com/status-im/status-desktop/assets/5157464/cf5c8425-dd16-4f69-acb6-442a0be1fe54

### CODE REVIEW

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -29,15 +29,15 @@ import shared.popups.send 1.0
 import shared.popups.send.views 1.0
 import shared.stores.send 1.0
 
+import StatusQ 0.1
+import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:35]: Import this as `SQUtils` instead, as done elsewhere

File: ui/imports/shared/stores/CommunityTokensStore.qml [hunk 2]
```diff
@@ -153,7 +153,7 @@ QtObject {
             root.setSignerStateChanged(communityId, communityName, status, url)
         }
 
-        function onOwnershipLost(communityId, communityName) {
+        function onOwnershipNodeLost(communityId, communityName) {
```
Comment (caybro) [inline ui/imports/shared/stores/CommunityTokensStore.qml:156]: How is this related? I see that it fixes something but you don't mention it anywhere. Put it at least in a separate commit

**Review by caybro — CHANGES_REQUESTED** [review]

File: ui/imports/shared/stores/CommunityTokensStore.qml (hunk 2 above)
Comment (Seitseman) [reply ui/imports/shared/stores/CommunityTokensStore.qml:156]: This error was logged in the console, just around the time the error with avatar has occurred. Its a real issue, but indeed not related to the avatar and ring bugs.
I'll put it in a separate commit.

File: ui/app/mainui/AppMain.qml (hunk 1 above)
Comment (Seitseman) [reply ui/app/mainui/AppMain.qml:35]: To be done

File: ui/app/mainui/AppMain.qml [hunk 3]
```diff
@@ -1967,7 +1967,7 @@ Item {
                 if (showQR.showSingleAccount || showQR.showForSavedAddress) {
                     return showQR.selectedAccount
                 }
-                return selectedReceiverAccount.item ?? ModelUtils.get(appMain.transactionStore.accounts, 0)
+                return selectedReceiverAccount.item ?? SQUtils.ModelUtils.get(appMain.transactionStore.accounts, 0)
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:1970]: OK, much better... but... how does a fix inside `ReceiveModal` have an impact on that user icon? CC @alexjba

File: ui/imports/shared/stores/CommunityTokensStore.qml (hunk 2 above)
Comment (caybro) [reply ui/imports/shared/stores/CommunityTokensStore.qml:156]: Yeah, this is good, here's the NIM signal:
```nim
proc ownershipNodeLost*(self: View, communityId: string, communityName: string) {.signal.}
```

Comment (Seitseman) [reply ui/imports/shared/stores/CommunityTokensStore.qml:156]: yes, I've found that signal on nim side and rename handler in qml accordingly

File: ui/app/mainui/AppMain.qml (hunk 3 above)
Comment (Seitseman) [reply ui/app/mainui/AppMain.qml:1970]: Here the root cause of the issue is in "Utils.qml" two files Utils.qml ui\imports\utils\Utils.qml and ui\StatusQ\src\StatusQ\Core\Utils\Utils.qml and recent changes introduced StatusQ's Utils.qml and it resulted in qtquick engine 'overwriting' previous and correct Utils.qml with one from StatusQ. Adding import qualifier resolves the problem

Comment (caybro) [reply ui/app/mainui/AppMain.qml:1970]: Alright... but it still doesn't explain my question :) Anyways, can you pls look into other occurrences of the two `Utils` being imported unaliased throughout the code? I'm pretty sure we have more cases like this

**Review by caybro — APPROVED** [review]

File: ui/StatusQ/src/StatusQ/Components/StatusChatList.qml [hunk 4]
```diff
@@ -4,7 +4,7 @@ import QtQuick.Controls 2.15 as QC
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Core.Utils 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
```
Comment (caybro) [inline ui/StatusQ/src/StatusQ/Components/StatusChatList.qml:7]: You don't need to do it inside StatusQ itself, you can't import the app's `utils` from here

File: ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml [hunk 5]
```diff
@@ -7,7 +7,7 @@ import StatusQ.Core 0.1
 import StatusQ.Controls 0.1
 import StatusQ.Components 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Core.Utils 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml:10]: Maybe it can be dropped if it's not used?

File: ui/app/AppLayouts/Chat/views/MembersEditSelectorView.qml [hunk 6]
```diff
@@ -4,7 +4,7 @@ import QtQuick.Layouts 1.14
 
 import StatusQ.Core 0.1
 import StatusQ.Core.Theme 0.1
-import StatusQ.Core.Utils 0.1
+import StatusQ.Core.Utils 0.1 as SQUtils
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/views/MembersEditSelectorView.qml:7]: Same here and elsewhere, if it's not used, remove it

**Review by caybro — COMMENTED** [review]
Looks good in general!

**Review by caybro — APPROVED** [review]

**Review by alexjba — APPROVED** [review]
LGTM!
